node/src/components/small_network.rs

```diff
@@ -435,6 +449,9 @@ where
                         times_seen_asymmetric: 0,
                     },
                 );
+                self.net_metrics
+                    .open_connections
+                    .set(self.incoming.len() as i64);
```

Comment:
What about outgoing connections? Should `self.outgoing.len()` be added here?
(Sorry if my questions don't make sense; I'm not familiar with the `small_network` code.)

Reply:
`small_network` currently has a 1:1 ratio of incoming to outgoing - if there is no matching outgoing connection for every incoming or vice verse, we attempt to rectify this. If it fails I believe we terminate both connections.

Reply:
I think it's probably ok as is.  We don't retain asymmetric connections for too long, so this will give a reasonable representation of the number of notional connections where each is comprised of two real connections.

Eventually I believe we'll either change to libp2p (where each connection is bidirectional) or we'll update small_network to have bidirectional connections, so this metric will remain consistent after we make that transition.